satisfactory_example/docker-compose.yml

```diff
@@ -0,0 +1,43 @@
+version: '3'
```

Comment:
Love the example, but what do you think about creating an examples folder to avoid cluttering root

so we'd have something like 
```
/
└── examples
    ├── minecraft
    └── satisfactory
```

Reply:
I think that's a great solution!